packages/grid/src/GridRenderer.ts

```diff
@@ -1085,8 +1085,7 @@ export class GridRenderer {
           columnX +
           i * treeDepthIndent +
           treeDepthIndent * 0.5 +
-          treeHorizontalPadding +
-          0.5;
+          treeHorizontalPadding;
```

Comment:
This `0.5` is here intentionally, for crisp lines. When drawing a 1 pixel line, you need to adjust it by half a pixel. https://stackoverflow.com/questions/9311428/draw-single-pixel-line-in-html5-canvas